src/main/resources/db/migration/local/afterMigrate.sql

```diff
@@ -23,3 +23,7 @@ SELECT -1, sg.grp_id FROM darts.security_group sg
 where sg.grp_id not in (select sgua.grp_id from darts.security_group_user_account_ae sgua where sgua.usr_id = -1)
 ORDER BY 2;
 
+
+update darts.automated_task
+set cron_expression = '0 0 1 * * *';
```

Comment:
Would it be less confusing, not to update everybody's local cron expressions?

Reply:
I know when i local test i only want to turn on the things that i want to test, so that automatedTasks that i am not testing do not fill my logs. 
If i want to test a specific cron job, then i can turn it on.
Do you think people test all automatedTasks every time they run their local instance?

Reply:
I like the idea of having the automated tasks "turned off" by default for local dev, as most of the time they won't be relevant to the ticket being worked on.

Reply:
It really doesn't bother me, though I prefer to become more familiar with my local postgres database with the expected settings. Knowing that I'm deviating from the norm myself only when I change things locally in a SQL query.  Otherwise I find it a hard time to keep up with knowing what "Normal" is supposed to look like on this project.  It also save me time having to look behind the meaning of these changes - looks like 1am.

```
aut_id	9
task_name	ProcessArmResponseFiles
task_description	Processes ARM response files
cron_expression	30 * * * * *
```
Anyway, I prefer that global changes like this get communicated, so that other devs are aware.
I suppose they should look at your PR though :)